.storybook/main.js

```diff
@@ -15,7 +15,7 @@ const PACKAGE_NAMES = readdirSync(path.resolve(__dirname, '../packages')).filter
 );
 
 module.exports = {
-  stories: ['../packages/*/demo/**/*.stories.@(js|jsx|ts|tsx|mdx)'],
+  stories: [`../packages/${process.env.PACKAGE || '*'}/demo/**/*.stories.@(js|jsx|ts|tsx|mdx)`],
```

Comment:
provides an efficiency mechanism for focused package development, i.e. `PACKAGE=accordions npm start` only compiles `accordions` stories.

Reply:
🤯 the update we needed